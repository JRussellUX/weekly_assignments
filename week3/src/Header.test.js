import Header from './Header.js';
import renderer from 'react-test-renderer';
import {render, screen} from '@testing-library/react';

test('renders header screen', () => {
  const header = renderer.create(<Header text="Test Header" />);
  let tree = header.toJSON();
  expect(tree).toMatchSnapshot();
});

it('Header appears', () => {
  render(<Header text="Test Header" />);
  const actualHeader = screen.getByText("Test Header");
  expect(actualHeader).toBeTruthy();
});