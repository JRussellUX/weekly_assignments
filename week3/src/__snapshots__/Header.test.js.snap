// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders header screen 1`] = `
<h1>
  Test Header
</h1>
`;
